README

```diff
@@ -8,3 +10,18 @@ To build the ding-libs from the tarball:
 To build the ding-libs from a source checkout:
     autoreconf -if
 then follow the tarball instructions above.
+
+# Requirements
+
+Build:
+
+- `autotools`
+- `gcc`
+- `libc`
+- `coreutils`
+
+Tests:
+
+- `file`
+- `gettext`
```

Comment:
I think it is a build dependency:
https://github.com/SSSD/ding-libs/blob/e5b5b1d44f476cfe406a86ec4600a13796361851/ini/ini_configobj.c#L33
etc

Reply:
Well it builds without it and without error. It's only at check time that things fail.

Reply:
Ah, I see, this is about command line tool from 'gettext-runtime', ok.

Reply:
Yep!